lib/solr/params.rb

```diff
@@ -9,10 +9,10 @@ class Params
     def initialize
       @parameters = {
         q: nil,
-        df: 'keywords_teim', # TODO: Is this actually the default we want? This class is used by the TermSearchAdapter too.
         fq: [],
         'facet.field'.to_sym => [],
-        start: 0
+        start: 0,
+        'facet.mincount': 1 # TODO: Check to make sure this is okay as a global default for digital objects and terms.
```

Comment:
Seems like the question is whether we want mincount = 1 or to use facet.missing - is the UI unusable if we keep the none facet values in the dropdowns?

Reply:
I don't entirely understand what `facet.missing` does. I have read the Solr documentation but it isn't making sense to me. Does it mean that if a facet doesn't have any values that correspond to the search, the facet field is returned but it is an empty array instead of being omitted all together?

Reply:
`facet.missing` returns a count for the items in the search that have no value for the facet, like a "no value" count. Which means that `facet.mincount` of 1 becomes meaningless, since there's always at least 1.

Reply:
`facet.mincount` of 1 just removes any facet values that don't appear in the search. That way we don't return a bunch of facet values that have a value of 0. I might be misunderstanding, but I don't think that would affect `facet.missing` if we decided to use that.

Reply:
Resolving since I was confused about what `facet.mincount` did!